website/docs/r/synapse_workspace_keys.html.markdown

```diff
@@ -6,7 +6,7 @@ description: |-
   Manages Synapse Workspace Keys
 ---
 
-# azurerm_synapse_workspace
+# azurerm_synapse_workspace_keys
```

Comment:
The resource name uses key instead of keys so this should be
```suggestion
# azurerm_synapse_workspace_key
```

Reply:
we should rename the file if so?